src/components/RelocateBoxForm/RelocateBoxForm.jsx

```diff
@@ -144,12 +153,32 @@ const RelocateBoxForm = ({ setFormSubmitted }) => {
             <FormErrorMessage>{errors.date?.message}</FormErrorMessage>
           </FormControl>
           <br />
-          <FormControl isInvalid={errors?.zipcode}>
+          <FormControl isInvalid={errors?.zipcode || errors['']?.message.startsWith('Postal code')}>
             <FormLabel htmlFor="zipcode" className={styles['required-field']}>
               Zip Code
             </FormLabel>
-            <Input id="zipCode" placeholder="e.g. 90210" name="zipcode" {...register('zipcode')} />
+            <Input id="zipcode" placeholder="e.g. 90210" name="zipcode" {...register('zipcode')} />
+            {/* display an error if there is no zipcode */}
             <FormErrorMessage>{errors.zipcode?.message}</FormErrorMessage>
+            {/* display an error if zipcode does not exist in country */}
+            {errors['']?.message !== 'zip validated' && (
+              <FormErrorMessage>{errors['']?.message}</FormErrorMessage>
+            )}
+          </FormControl>
+          <br />
+          <FormControl isInvalid={errors?.country}>
+            <FormLabel htmlFor="country" className={styles['required-field']}>
+              Country
+            </FormLabel>
+            <Controller
+              control={control}
+              name="country"
+              // eslint-disable-next-line no-unused-vars
+              render={({ field: { onChange, value, ref } }) => (
+                <ChakraReactSelect options={countryOptions} value={value} onChange={onChange} />
```

Comment:
This applies to all of the country dropdowns:
Currently, when the user opens the dropdown, the dropdown pushes the rest of the content on the form down so the message field gets pushed to below the last element in the dropdown. Is it possible to change this such that the dropdown is instead covering the message field when the user opens it? This is how I've usually seen dropdowns in forms so I think that would be nice to have here as well.

For example, if you take a look at the dropdowns [here](https://getbootstrap.com/docs/4.3/components/dropdowns/), you'll notice that when the user opens the dropdowns, the rest of the content on the page remains in the same place and the dropdown items simply cover whatever is behind it. Let me know if you need more clarification on this.

Reply:
Resolving this since this is not an actual issue - the page extends since the dropdown needs more space to show the items as discussed in DMs.